### DISCUSSION TITLE
critical flame tempreature and autoignition temp

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#13701] [Q&A]
Hello team,

I’m using FDS 6.7.1 to conduct water mist tests on heptane pool fires. Based on my understanding, increasing the autoignition temperature (AIT) and critical flame temperature (CFT) in the reaction generally reduces the extinguishing time. However, when I adjusted the CFT from the default 1437°C to 1457°C, the extinguishing time actually increased, even though all other parameters remained the same in both cases.

Could you provide any insights into why this might be happening?

I also have another question. I know we should monitor the CFL number to assess whether the mesh is sufficiently fine. Are there any other methods to confirm that my mesh size is appropriate? For reference, my fire size is 2.4 MW, and I’m using four nozzles with a 60 bar, K1.54 water mist system.

Thank you, and have a great day!

**Comment by drjfloyd** [comment]
The CFL doesn't tell you anything about the suitability of the mesh size. The CFL is an input that FDS uses to determine what time step to take based on the conditions in the simulation. A grid study would be one option.

If you are using the predefined species N-HEPTANE, then per the User's Guide the assumed CFT for N-HEPTANE is 1497 so setting 1457 is a reduction in CFT which would be expected to have longer times to extinction.

**Comment by HunterXHunter1997** [comment]
Thanks for the quick response.
Do you recommend using the latest FDS version "Will it make a difference"?
and by the way here is my model:


&HEAD CHID='redo_4x5_1_54K_60bar__(200AIT)_1457'/
&TIME T_END=900.0/
&DUMP RENDER_FILE='redo_4x5_1_54K_60bar__(200AIT)_1457.ge1', COLUMN_DUMP_LIMIT=.TRUE., DT_RESTART=300.0, DT_SL3D=0.25/
&MISC TMPA=22.3/

&MESH ID='Mesh01-a-merged-merged-a', IJK=9,12,12, XB=0.0,2.25,0.0,3.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-b', IJK=9,12,9, XB=0.0,2.25,0.0,3.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-c', IJK=9,12,11, XB=0.0,2.25,0.0,3.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-d', IJK=9,14,12, XB=0.0,2.25,3.0,6.5,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-e', IJK=9,14,9, XB=0.0,2.25,3.0,6.5,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-f', IJK=9,14,11, XB=0.0,2.25,3.0,6.5,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-g', IJK=9,14,12, XB=0.0,2.25,6.5,10.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-h', IJK=9,14,9, XB=0.0,2.25,6.5,10.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-i', IJK=9,14,11, XB=0.0,2.25,6.5,10.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-j', IJK=11,12,12, XB=2.25,5.0,0.0,3.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-k', IJK=11,12,9, XB=2.25,5.0,0.0,3.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-l', IJK=11,12,11, XB=2.25,5.0,0.0,3.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-m', IJK=11,14,12, XB=2.25,5.0,3.0,6.5,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-n', IJK=11,14,9, XB=2.25,5.0,3.0,6.5,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-o', IJK=11,14,11, XB=2.25,5.0,3.0,6.5,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-p', IJK=11,14,12, XB=2.25,5.0,6.5,10.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-q', IJK=11,14,9, XB=2.25,5.0,6.5,10.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-r', IJK=11,14,11, XB=2.25,5.0,6.5,10.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-s', IJK=10,12,12, XB=5.0,7.5,0.0,3.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-t', IJK=10,12,9, XB=5.0,7.5,0.0,3.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-u', IJK=10,12,11, XB=5.0,7.5,0.0,3.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-v', IJK=40,56,48, XB=5.0,7.5,3.0,6.5,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-w', IJK=10,14,9, XB=5.0,7.5,3.0,6.5,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-x', IJK=10,14,11, XB=5.0,7.5,3.0,6.5,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-y', IJK=10,14,12, XB=5.0,7.5,6.5,10.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-z', IJK=10,14,9, XB=5.0,7.5,6.5,10.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-{', IJK=10,14,11, XB=5.0,7.5,6.5,10.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-|', IJK=10,12,12, XB=7.5,10.0,0.0,3.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-}', IJK=10,12,9, XB=7.5,10.0,0.0,3.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-~', IJK=10,12,11, XB=7.5,10.0,0.0,3.0,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-', IJK=10,14,12, XB=7.5,10.0,3.0,6.5,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-', IJK=10,14,9, XB=7.5,10.0,3.0,6.5,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-', IJK=10,14,11, XB=7.5,10.0,3.0,6.5,5.25,8.0/
&MESH ID='Mesh01-a-merged-merged-', IJK=10,14,12, XB=7.5,10.0,6.5,10.0,0.0,3.0/
&MESH ID='Mesh01-a-merged-merged-', IJK=10,14,9, XB=7.5,10.0,6.5,10.0,3.0,5.25/
&MESH ID='Mesh01-a-merged-merged-', IJK=10,14,11, XB=7.5,10.0,6.5,10.0,5.25,8.0/

&SPEC ID='WATER VAPOR'/

&PART ID='igniter particle',
      STATIC=.TRUE.,
      SURF_ID='igniter surface',
      AGE=60.0/
&PART ID='Nozzles water',
      SPEC_ID='WATER VAPOR',
      DIAMETER=163.6,
      MINIMUM_DIAMETER=76.3,
      MAXIMUM_DIAMETER=400.4,
      DISTRIBUTION='ROSIN-RAMMLER',
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE DIAMETER','PARTICLE VELOCITY','PARTICLE AGE',
      AGE=60.0,
      SAMPLING_FACTOR=1/
&PART ID='Water',
      SPEC_ID='WATER VAPOR',
      DIAMETER=500.0,
      MONODISPERSE=.TRUE.,
      AGE=60.0,
      SAMPLING_FACTOR=1/

&REAC ID='Heptane',
      FUEL='N-HEPTANE',
      CRITICAL_FLAME_TEMPERATURE=1457.0,
      AUTO_IGNITION_TEMPERATURE=200.0,
      CO_YIELD=6.0E-3,
      SOOT_YIELD=0.015,
      HEAT_OF_COMBUSTION=4.0E4,
      RADIATIVE_FRACTION=0.35/

&RAMP ID='Nozzles control_RAMP', T=-0.25, F=1.0/
&RAMP ID='Nozzles control_RAMP', T=0.25, F=-1.0/
&RAMP ID='Nozzles control_RAMP', T=14.75, F=-1.0/
&RAMP ID='Nozzles control_RAMP', T=15.25, F=1.0/
&RAMP ID='Nozzles control_RAMP', T=1619.75, F=1.0/
&RAMP ID='Nozzles control_RAMP', T=1620.25, F=-1.0/
&PROP ID='Generic Commercial Spray01',
      PART_ID='Nozzles water',
      OFFSET=0.0,
      PARTICLES_PER_SECOND=10000,
      K_FACTOR=1.54,
      OPERATING_PRESSURE=60.0,
      SPRAY_PATTERN_TABLE='Generic Commercial Spray01_SPRAY_PATTERN_TABLE'/
&TABL ID='Generic Commercial Spray01_SPRAY_PATTERN_TABLE', TABLE_DATA=0.0,30.0,0.0,60.0,90.0,0.2/
&TABL ID='Generic Commercial Spray01_SPRAY_PATTERN_TABLE', TABLE_DATA=0.0,30.0,72.0,132.0,90.0,0.2/
&TABL ID='Generic Commercial Spray01_SPRAY_PATTERN_TABLE', TABLE_DATA=0.0,30.0,144.0,204.0,90.0,0.2/
&TABL ID='Generic Commercial Spray01_SPRAY_PATTERN_TABLE', TABLE_DATA=0.0,30.0,216.0,276.0,90.0,0.2/
&TABL ID='Generic Commercial Spray01_SPRAY_PATTERN_TABLE', TABLE_DATA=0.0,30.0,288.0,348.0,90.0,0.2/
&CTRL ID='Nozzles control', FUNCTION_TYPE='CUSTOM', RAMP_ID='Nozzles control_RAMP', LATCH=.FALSE., INPUT_ID='TIME'/
&DEVC ID='NOZZLE1', PROP_ID='Generic Commercial Spray01', XYZ=2.5,3.0,7.7, LATCH=.FALSE., QUANTITY='CONTROL', CTRL_ID='Nozzles control'/
&DEVC ID='NOZZLE2', PROP_ID='Generic Commercial Spray01', XYZ=2.5,7.0,7.7, LATCH=.FALSE., QUANTITY='CONTROL', CTRL_ID='Nozzles control'/
&DEVC ID='NOZZLE4', PROP_ID='Generic Commercial Spray01', XYZ=7.5,7.0,7.7, LATCH=.FALSE., QUANTITY='CONTROL', CTRL_ID='Nozzles control'/
&DEVC ID='NOZZLE3', PROP_ID='Generic Commercial Spray01', XYZ=7.5,3.0,7.7, LATCH=.FALSE., QUANTITY='CONTROL', CTRL_ID='Nozzles control'/
&DEVC ID='O2-3 (mass fraction)', QUANTITY='MASS FRACTION', SPEC_ID='OXYGEN', XYZ=6.254834,4.093544,0.9/
&DEVC ID='T1', QUANTITY='THERMOCOUPLE', XYZ=5.207838,5.355114,1.702504/
&DEVC ID='T2', QUANTITY='THERMOCOUPLE', XYZ=4.996453,5.055114,3.348609/
&DEVC ID='T5', QUANTITY='THERMOCOUPLE', XYZ=6.334671,4.755114,0.9/
&DEVC ID='T7', QUANTITY='THERMOCOUPLE', XYZ=5.748748,4.755114,0.9/
&DEVC ID='T8', QUANTITY='THERMOCOUPLE', XYZ=6.334671,5.255114,0.9/
&DEVC ID='O2-1 (mass fraction)', QUANTITY='MASS FRACTION', SPEC_ID='OXYGEN', XYZ=4.403476,0.349126,1.0/
&DEVC ID='O2-2 (mass fraction)', QUANTITY='MASS FRACTION', SPEC_ID='OXYGEN', XYZ=4.403476,1.603632,1.0/
&DEVC ID='FM1-TREE 4,5m', QUANTITY='THERMOCOUPLE', XYZ=5.013161,5.055114,4.5/
&DEVC ID='FM2-TREE 6m', QUANTITY='THERMOCOUPLE', XYZ=5.013161,5.055114,6.0/
&DEVC ID='FM3-TREE 9m', QUANTITY='THERMOCOUPLE', XYZ=5.013161,5.055114,7.5/
&DEVC ID='O2-2 (volume fraction)', QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', XYZ=4.403476,1.603632,1.0/
&DEVC ID='O2-1 (volume fraction)', QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', XYZ=4.403476,0.349126,1.0/
&DEVC ID='O2-3 (volume fraction)', QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', XYZ=6.254834,4.093544,0.9/
&DEVC ID='O2-2 (TEMP)', QUANTITY='THERMOCOUPLE', XYZ=4.403476,1.603632,1.0/
&DEVC ID='O2-1 (TEMP)', QUANTITY='THERMOCOUPLE', XYZ=4.403476,0.349126,1.0/
&DEVC ID='O2-3 (TEMP)', QUANTITY='THERMOCOUPLE', XYZ=6.254834,4.093544,0.9/
&DEVC ID='TIME', QUANTITY='TIME', XYZ=0.0,0.0,0.0/

&MATL ID='STEEL (PAN)',
      SPECIFIC_HEAT=0.46,
      CONDUCTIVITY=45.8,
      DENSITY=7850.0/
&MATL ID='heptane',
      SPECIFIC_HEAT=2.24,
      CONDUCTIVITY=0.14,
      DENSITY=675.0,
      ABSORPTION_COEFFICIENT=187.5,
      EMISSIVITY=1.0,
      HEAT_OF_REACTION=317.0,
      SPEC_ID(1,1)='N-HEPTANE',
      NU_SPEC(1,1)=1.0,
      BOILING_TEMPERATURE=98.35/

&SURF ID='igniter surface',
      COLOR='RED',
      HRRPUA=2000.0,
      RAMP_Q='igniter surface_RAMP_Q',
      TMP_FRONT=2000.0,
      GEOMETRY='SPHERICAL',
      RADIUS=0.1/
&RAMP ID='igniter surface_RAMP_Q', T=0.0, F=1.0/
&RAMP ID='igniter surface_RAMP_Q', T=10.0, F=1.0/
&RAMP ID='igniter surface_RAMP_Q', T=11.0, F=0.0/
&RAMP ID='igniter surface_RAMP_Q', T=1800.0, F=0.0/
&SURF ID='steel',
      COLOR='BLACK',
      BACKING='VOID',
      MATL_ID(1,1)='STEEL (PAN)',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=1.73E-3/
&SURF ID='heptane',
      COLOR='WHITE',
      BACKING='VOID',
      MATL_ID(1,1)='heptane',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.12/

&INIT ID='IGNITER', PART_ID='igniter particle', N_PARTICLES=1, XYZ=6.14356,4.977854,0.9/ 

&OBST ID='Obstruction', XB=5.498748,6.498748,4.5,5.5,0.694853,0.794118, SURF_IDS='heptane','steel','steel'/ 
&OBST ID='Obstruction', XB=5.498748,6.498748,4.5,4.5,0.794118,0.893382, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=5.498748,6.498748,5.5,5.5,0.794118,0.893382, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=5.498748,5.498748,4.5,5.5,0.794118,0.893382, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=6.498748,6.498748,4.5,5.5,0.794118,0.893382, SURF_ID='steel'/ 

&VENT ID='Vent', SURF_ID='OPEN', XB=4.0,6.0,0.0,0.0,0.5,2.5/ 

&SLCF QUANTITY='TEMPERATURE', PBX=4.46/
&SLCF QUANTITY='TEMPERATURE', PBX=5.699338/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', PBY=5.95213/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', PBY=9.45213/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', PBY=12.95213/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', PBX=4.5/
&SLCF QUANTITY='TEMPERATURE', XB=0.0,10.0,0.0,10.0,0.0,8.0, FYI='Temperature'/
&SLCF QUANTITY='VELOCITY', XB=0.0,10.0,0.0,10.0,0.0,8.0, FYI='velocity'/
&SLCF QUANTITY='MPUV', PART_ID='Water', XB=0.0,10.0,0.0,10.0,0.0,8.0, FYI='water '/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', XB=0.0,10.0,0.0,10.0,0.0,8.0, FYI='water vapor'/

&DEVC ID='[Particle: Water] Accumulated Mass Per Unit Area_SURFACE INTEGRAL', QUANTITY='AMPUA', PART_ID='Water', SPATIAL_STATISTIC='SURFACE INTEGRAL', XB=0.0,10.0,0.0,10.0,0.0,8.0/
&DEVC ID='[Species: WATER VAPOR] Volume Fraction_MASS MEAN', QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', SPATIAL_STATISTIC='MASS MEAN', XB=0.0,10.0,0.0,10.0,0.0,8.0/
&DEVC ID='[Species: WATER VAPOR] Volume Fraction_VOLUME MEAN', QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', SPATIAL_STATISTIC='VOLUME MEAN', XB=0.0,10.0,0.0,10.0,0.0,8.0/
&DEVC ID='[Species: WATER VAPOR] Volume Fraction_MAX', QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', SPATIAL_STATISTIC='MAX', XB=0.0,10.0,0.0,10.0,0.0,8.0/
&DEVC ID='[Species: WATER VAPOR] Volume Fraction_MEAN', QUANTITY='VOLUME FRACTION', SPEC_ID='WATER VAPOR', SPATIAL_STATISTIC='MEAN', XB=0.0,10.0,0.0,10.0,0.0,8.0/


&TAIL /

**Comment by mcgratta** [comment]
You should always use the latest version of FDS, especially for cases like this that have features that are not robust. Extinction and sprays are evolving. If you ask a question based on some code from however many years ago, we will not be able to help you.

**Reply by HunterXHunter1997** [reply]
Thanks mcgratta
Ok I downloaded fds 6.9.1 but when I launched my diesel spray simulation the stopped because of numircal unstaibility after the diesel spray starts. 
By the way in fds 6.7.1 it didnt show this kind of warnings. 
Do you have an idea what I should change there ? 
I am using a predefined dodecane in my case

**Comment by mcgratta** [comment]
This case is complicated. I suggest that you create a simple case with one mesh and just the spray nozzle and near surroundings. With all these very small meshes, there might be a problem related to particles crossing boundaries.

**Reply by HunterXHunter1997** [reply]
Ok, can you please suggest studies in the validation guide that are near to my cases:
1-Heptane pool fire (water mist suppression).
2-Diesel spray with diesel pool fire (water mist suppression).

**Reply by mcgratta** [reply]
USCG/HAI -- mist suppression experiments conducted by Hughes Associates on behalf of US Coast Guard.
